index.bs

```diff
@@ -504,6 +505,67 @@ QuicTransportBase includes DatagramTransport;
 QuicTransportBase includes WebTransport;
 </pre>
 
+## Methods ##  {#quic-transport-base-methods}
+
+: <dfn for="QuicTransportBase" method>getStats()</dfn>
+:: Gathers stats for this <code><a>QuicTransportBase</a></code>'s QUIC
+   connection and reports the result asynchronously.</p>
+
+   When close is called, the user agent MUST run the following steps:
+     1. Let |transport| be the QuicTransportBase on which `getStats` is invoked.
+     1. If |transport|'s {{[[WebTransportState]]}} is `"new"`, then abort
+        these steps.
+     1. Let |p| be a new promise.
+     1. Return |p| and continue the following steps in background.
+         1. Gather the stats from the underlying QUIC connection.
+         1. Once stats have been gathered, resolve |p| with the
+            {{QuicTransportStats}} object, representing the gathered stats.
+
+## `QuicTransportStats` Dictionary ##  {#quic-transport-stats}
+
+The <dfn dictionary>QuicTransportStats</dfn> dictionary includes information
+on QUIC connection level stats.
+
+<pre class="idl">
+dictionary QuicTransportStats {
+  DOMHighResTimeStamp timestamp;
+  unsigned long long bytesSent;
+  unsigned long long packetsSent;
+  unsigned long numOutgoingStreamsCreated;
+  unsigned long numIncomingStreamsCreated;
+  unsigned long long bytesReceived;
+  unsigned long long packetsReceived;
+  DOMHighResTimeStamp minRttUs;
```

Comment:
Does this need the "Us" suffix since it has the DOMHighResTimeStamp type?  "timestamp" doesn't have it.